service/siteService.test.js

```diff
@@ -1,16 +1,119 @@
-const Airtable = require('airtable')
-
+const airtableMock = require('airtable')
+const airtableClientMock = require('../airtableClient')
+const cacheServiceMock = require('./cacheService')
 const siteService = require('./siteService')
 
-describe("siteService tests", () => {
-    test('transforming list of public transit option strings returns list of publicTransitOption objects', () => {
+jest.mock('airtable')
+jest.mock('../airtableClient')
+jest.mock('./cacheService')
+
+describe("getMutualAidSites", () => {
```

Comment:
Since you now have a set of try/catch blocks, it's worth writing two more tests (with associated mocks) to make sure they work as expected.  Specifically:

* When `airtable.getMutualAidSites()` throws an exception
* When something outside of that `async` throws an exception (The easiest thing to mock is probably `cacheService.writeCache()` on L34, but the quickest would be `console.log()` on L22

Reply:
😭  you're right

Reply:
:comfort: